data_architect_misc/data_transformer/transform_functions/common_transform_functions.py

```diff
@@ -224,10 +224,8 @@ def drop_empty_rows(
                                                       "names being string values.")
         for col_name in list_of_col_names:
             df = df[df[col_name].astype(bool)]
-
```

Comment:
Usually, I leave empty line between for, if and return statements. That way, it's easier for a human eye to catch them as a unit/group. That's okay to remove them this time, but in the future, please don't assume that the empty lines are to be removed. They are there for a reason. :)